src/expressions.rst

```diff
@@ -2118,7 +2121,7 @@ Destructuring Assignment
 :dp:`fls_2eheo4yo2orm`
 A :t:`destructuring assignment` is an :t:`assignment expression` where
 the :t:`assignee operand` is either an :t:`array expression`, a :t:`struct
-expression`, or a :t:`tuple expression`.
+expression`, a :t:`tuple struct call expression` or a :t:`tuple expression`.
```

Comment:
Here comes the alphabetization maniac:

expression, a tuple expression, or a tuple struct call expression.